src/style.css

```
@@ -1 +1,82 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 300;
+  font-style: normal;
+}
+
+.body {
+  font-family: Avenir, sans-serif;
+  display: flex;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  justify-content: start;
+  align-items: flex-start;
+  height: 100vh;
+  width: 100%;
+}
+
+.search-panel {
+  position: relative;
+  width: 100%;
+  margin: 20px 8px;
+}
+
+.search__input {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  width: 100%;
+  font-size: 16px;
+  border-radius: 3px;
+  box-sizing: border-box;
+  transition: all 0.3s ease;
+  gap: 20px;
+  border: 1px solid #ccc;
+}
+
+.search__input--big {
+  background-position: 25px 26px;
+  background-size: 19px 19px, 100% 100%;
+  padding-top: 25px;
+  padding-left: 62px;
+  padding-bottom: 26px;
+  width: 100%;
+  height: 70px;
+  opacity: 1;
+  border: 1px solid #e1e7ed;
+  color: #687C94;
+  background-color: #fff;
+}
+
+.search__input--small {
+  background-position: 15px 13px;
+  background-size: 11px 11px;
+  padding-top: 12px;
+  font-size: 14px;
+  padding-left: 34px;
+  font-weight: 300px;
```

```suggestion
  font-weight: 300;
```
px is wrong unit for font-weight